frontend/src/components/Arena/Arena.jsx

```diff
@@ -42,6 +55,14 @@ export default function Arena() {
       });
   };
 
+  const minute = () => {
+    return Math.floor(time / 60);
+  };
```

Comment:
time should be a parameter, and this function should be outside the component (not related to react)